### PR TITLE
Add a _redirects file

### PR DESCRIPTION
**Pull request by stevelikesmusic**
## Description

This file allows netlify to rewrite 404 requests to always serve index.html. Doing
so allows the single page app to work. Requests to a path like /add work if the
user clicks to them. But if the browser loads that path, netlify doesn't find a file
named add.html and returns a 404.

The redirects file tells the file server to always
return index.html for any request—which is what we want since everything is client
driven.

I noticed when QA-ing the last couple PRs that netlify was returning 404s when I refreshed the page rather than serving up the entry file and letting client side routing handle everything. We should probably include this in the project template @segdeha.

## Type of Changes

|     | Type                       |
| --- | -------------------------- |
|    | :bug: Bug fix              |
|  ✓ | :sparkles: New feature     |
|    | :hammer: Refactoring       |
|    | :100: Add tests            |
|    | :link: Update dependencies |
|    | :scroll: Docs              |

## Testing Steps / QA Criteria

1. Go to `/add`
1. Hit refresh
1. Everything still works 💰

### CODE REVIEW

File: _redirects [hunk 1]
```diff
@@ -0,0 +1 @@
+/*    /index.html   200
```
Comment (segdeha) [inline _redirects]: Easy peasy! Consider adding a trailing `\n`, otherwise this looks 👍

**Review by segdeha — APPROVED** [review]
Thanks for doing this!

**Comment by segdeha** [comment]
@stevelikesmusic Would you mind opening a PR to add this to the project template, too?

**Comment by laurenmbeatty** [comment]
@segdeha is it working for you?  I'm still getting a "Page not Found" message.

**Comment by stevelikesmusic** [comment]
That was me @laurenmbeatty I put the file in the wrong place 😂

File: _redirects (hunk 1 above)
Comment (stevelikesmusic) [reply _redirects]: Where are you Prettier

**Pull request merged**